plugins/CoreHome/tests/Integration/LoginAllowlistTest.php

```diff
@@ -132,7 +132,13 @@ public function test_getAllowlistedLoginIps_shouldReturnIpsAndTrimIfNeeded()
     public function test_getAllowlistedLoginIps_shouldResolveIp()
     {
         $this->setGeneralConfig('login_allowlist_ip', ['192.168.33.1', 'matomo.org', '127.0.0.1']);
-        $this->assertSame(['192.168.33.1', '185.31.40.177', '127.0.0.1'], $this->allowlist->getAllowlistedLoginIps());
+        $this->assertSame(['192.168.33.1', '185.31.40.177', '2a00:b6e0:1:200:177::1', '127.0.0.1'], $this->allowlist->getAllowlistedLoginIps());
+    }
+
+    public function test_getAllowlistedLoginIps_shouldResolveIpv6Only()
+    {
+        $this->setGeneralConfig('login_allowlist_ip', ['192.168.33.1', 'integration-test.matomo.org', '127.0.0.1']);
+        $this->assertSame(['192.168.33.1', 'integration-test.matomo.org', '::1', '127.0.0.1'], $this->allowlist->getAllowlistedLoginIps());
```

Comment:
it may not be expected that it returns the integration test hostname here, but it was already doing it before so it shouldn't be a problem. Only fixing the IPv6 part for now